lib/decrypt.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/enigma'
+
+enigma = Enigma.new
+key = ARGV[2]
+date = ARGV[3]
```

Comment:
ARGV[2] is the third argument passed in the command line, which should be the key. ARGV[3] should be the date